src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -41,11 +45,17 @@ public OverseasEntitySubmissionDtoValidator(EntityDtoValidator entityDtoValidato
     public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
         entityDtoValidator.validate(overseasEntitySubmissionDto.getEntity(), errors, loggingContext);
         presenterDtoValidator.validate(overseasEntitySubmissionDto.getPresenter(), errors, loggingContext);
-        if(Objects.nonNull(overseasEntitySubmissionDto.getOverseasEntityDueDiligence())) {
-            overseasEntityDueDiligenceValidator.validate(overseasEntitySubmissionDto.getOverseasEntityDueDiligence(), errors, loggingContext);
-        }
-        if(Objects.nonNull(overseasEntitySubmissionDto.getDueDiligence())) {
-            dueDiligenceValidator.validate(overseasEntitySubmissionDto.getDueDiligence(), errors, loggingContext);
+
+        DueDiligenceDto dueDiligenceDto = overseasEntitySubmissionDto.getDueDiligence();
+        OverseasEntityDueDiligenceDto overseasEntityDueDiligenceDto = overseasEntitySubmissionDto.getOverseasEntityDueDiligence();
+        if (DueDiligenceOptionValidators.bothDueDiligenceOptionsNotSupplied(dueDiligenceDto, overseasEntityDueDiligenceDto, errors, loggingContext)) {
```

Comment:
We need to check that only one of the DueDiligence is present we can not have both and we can not allow both empty, last one has been covered.